alb-service/main.tf

```diff
@@ -0,0 +1,233 @@
+/**
+ * The web-service is similar to the `service` module, but the
+ * it provides a __public__ ALB instead.
+ *
+ * Usage:
+ *
+ *      module "auth_service" {
+ *        source    = "github.com/segmentio/stack/service"
```

Comment:
wait, isn't this wrong here _and_ in `stack/master/web-service/main.tf`? Seems like this line should be:
```
source    = "github.com/segmentio/stack/alb-service"
```
no?